content/en/docs/release_notes/villasimius.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Villasimius release notes
+linkTitle: Villasimius
+description: 9 September 2022
+date: 2022-09-09
+Hide_readingtime: true
+---
+## Summary
+Brief summary of the notable release contents.
+
+{{% releasenotes/upgrade %}}
+
+<!-- ## Breaking changes -->
+
+<!-- ## Features -->
+
+## Fixes
+* #7500: Fixed the unexpected re-rendering of UI components in the API DOCS & TEST page after a restart.
```

Comment:
**nitpick**: We've never capitalized like that, and it's "API Doc & Test":

```suggestion
* #7500: Fixed the unexpected re-rendering of UI components in the _API Doc & Test_ page after a restart.
```

Reply:
No idea what happened right here.. I agree, fixed.